concordium-sdk/src/main/java/com/concordium/sdk/crypto/wallet/web3Id/CredentialAttribute.java

```diff
@@ -0,0 +1,77 @@
+package com.concordium.sdk.crypto.wallet.web3Id;
+
+import java.io.IOException;
+
+import com.concordium.sdk.crypto.wallet.web3Id.CredentialAttribute.CredentialAttributeType;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@JsonSerialize(using = CredentialAttributeTypeSerializer.class)
+@JsonDeserialize(using = CredentialAttributieTypeDeserializer.class)
+@Builder
+@Getter
+public class CredentialAttribute {
+    enum CredentialAttributeType {
+        INT,
+        STRING,
+        TIMESTAMP;
+    }
+
+    private String value;
+    private CredentialAttributeType type;
+
+}
+
+class CredentialAttributieTypeDeserializer extends JsonDeserializer<CredentialAttribute> {
```

Comment:
Either place these in separate files or make them static subclasses of `CredentialAttribute`.

Reply:
I assume you mean inner classes, in which I agree/will do that.